src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -120,6 +120,8 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*smoke_FakeQuantizeLayerCPUTest.*bin.*)",
         // Issue: 69222
         R"(.*smoke_PriorBoxClustered.*PriorBoxClusteredLayerCPUTest.*_netPRC=f16_.*)",
+        // Skip the PriorBox from opset8 due to this is not supported by plugin and executed on reference.
+        R"(.*smoke_PriorBox_Basic/PriorBoxLayerTest.*)",
```

Comment:
Does it really save any sufficient amount of time in bounds of func test run? If no I would still prefer to keep the test given it covers fallback on reference impl path.

Reply:
They last about 1 second.

Reply:
Doesn't looks like significant value. I still would prefer to test fallback on referenec codepath. 
We can descrease number of instances if required.

Reply:
Doesn't looks like significant value. I still would prefer to test fallback on reference codepath. 
We can descrease number of instances if required.